load_data.js

```diff
@@ -0,0 +1,52 @@
+(function(window){
+  window.extractData = function() {
+    var ret = $.Deferred();
+
+    FHIR.oauth2.ready(function(smart){
+      var patient = smart.patient;
+      var pt = patient.read();
+     
+      $.when(pt).done(function(patient){
+		var gender = patient.gender;
+		var dob = new Date(patient.birthDate);		 
+		var day = dob.getDate();
+		var monthIndex = dob.getMonth();
+		var year = dob.getFullYear();
+
+		var dobStr = monthIndex + '/' + day + '/' + year;
+		
+		var fname = patient.name[0].given.join(" ");
+		var lname = patient.name[0].family.join(" ");
+		var age = calculateAge(dob);
+		
+		p = defaultPatient();
+		p.birthday = {value:dobStr};
+		p.gender={value:gender};
+		p.givenName={value:fname};
+		p.familyName={value:lname};
+		p.age = {value:age};
+		ret.resolve(p);
+		
+      });
+    });
+    return ret.promise();
+  };
+  
+  function calculateAge(birthday) { // birthday is a date

```

Comment:
Might want to consume the compute age method from here : https://github.com/cerner/cardiac-risk-app/blob/ascvd/build/js/cardiac_risk.js#L204-L226  since it takes care of leap years as well.